### PR TITLE
Feat/manager 6017

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-6017
| License          | BSD 3-Clause

## Description

Goals of this PR it to enforce IPV6 usage except if CDN is deactivated.

### CODE REVIEW

**Review by marie-j — DISMISSED** [review]

File: packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.controller.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ export default class MultisiteDomainConfigurationCtrl {
 
   onCDNChange(isCDNActive) {
     this.cdn = isCDNActive ? CDN_VALUES.ACTIVE : CDN_VALUES.NONE;
+    this.ipv6 = !isCDNActive;
```
Comment (JeremyDec) [inline packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.controller.js:31]: This doesn't seems to be used. Also the method `onCDNChange` seems to be useless then

Comment (mohammed-zahaf) [reply packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.controller.js:31]: It's used to display (ipV6: enable) when the cdn is deactivated.

Comment (mohammed-zahaf) [reply packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.controller.js:31]: it's bind like this: ipv6="selected.domain.ipV6Enabled"

Comment (JeremyDec) [reply packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.controller.js:31]: Ok seen together

**Review by JeremyDec — APPROVED** [review]

**Review by marie-j — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

File: packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.html [hunk 2]
```diff
@@ -36,16 +36,6 @@
     label="{{:: 'hosting_multisite_domain_configuration_options_choose' | translate }}"
     size="l"
 >
-    <oui-checkbox name="ipv6" model="$ctrl.ipv6">
-        <span
-            data-translate="hosting_multisite_domain_configuration_ipv6"
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.html:41]: If both translation keys are no more used, then you can also removed them from:
- `packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/translations/Messages_fr_FR.json`.

**Review by antleblanc — APPROVED** [review]

File: packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.html (hunk 2 above)
Comment (mohammed-zahaf) [reply packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.html:41]: i remember that we discuss about a tool which list all no used tran. keys, right ?
Can we use it to remove keys or not ?

Comment (mohammed-zahaf) [reply packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.html:41]: also how can we be sure the key is not used in split mode ?
Like: 'hosting_tab_DOMAINS_configuration_add_step2_mode_' + selected.mode + '_information'

Comment (mohammed-zahaf) [reply packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.html:41]: I removed all non used keys

**Review by marie-j — DISMISSED** [review]

**Comment by marie-j** [comment]
@mohammed-zahaf Could you please solve conflicts ? 
Thanks

**Comment by mohammed-zahaf** [comment]
> @mohammed-zahaf Could you please solve conflicts ?
> Thanks

Hello @marie-j , i resolved them thanks.

File: packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.html (hunk 2 above)
Comment (antleblanc) [reply packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.html:41]: > i remember that we discuss about a tool which list all no used tran. keys, right ?
> Can we use it to remove keys or not ?

Not implemented yet!

**Review by antleblanc — APPROVED** [review]

**Pull request merged**